### PR TITLE
Get Daily History - Android

### PR DESCRIPTION
**Pull request by GGGava**
Screenshot 2022-10-13 at 15 51 41
Screenshot 2022-10-13 at 15 52 54

### CODE REVIEW

File: android/src/main/java/co/areyouonpoint/pointsdk/capacitor/Mappers.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            metrics.map { put(it.toResponse()) }
+        })
+    }
+
+fun HealthMetric.toResponse() =
+    JSObject().apply {
+        putSafe("type", type.rawValue)
+        putSafe("date", date)
+        putSafe("value", value)
+        putOpt("variance", variance)
```
Comment (ae-ios-devs) [inline android/src/main/java/co/areyouonpoint/pointsdk/capacitor/Mappers.kt:87]:
```suggestion
        putOpt("variance", variance ?: JSONObject.NULL)
```

File: android/src/main/java/co/areyouonpoint/pointsdk/capacitor/Mappers.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        putOpt("workoutId", workoutId)
```
Comment (ae-ios-devs) [inline android/src/main/java/co/areyouonpoint/pointsdk/capacitor/Mappers.kt:88]:
```suggestion
        putOpt("workoutId", workoutId ?: JSONObject.NULL)
```

**Comment by AnnyFigueira** [comment]
Please take a new screenshot or paste the jsons after applying the suggestions made by Felipe please

**Comment by GGGava** [comment]
Screenshot 2022-10-14 at 11 12 56

**Review by AnnyFigueira — APPROVED** [review]

**Review by ae-ios-devs — APPROVED** [review]

**Pull request merged**